tutorials/nbconverted/general_usage.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Pycytominer General Usage Walkthrough!
+# 
+# Welcome to this walkthrough where we will guide you through the process of extracting single cell morphology features using the [`pycytominer`](https://github.com/cytomining/pycytominer) API.
+# 
+# For this walkthrough, we will be working with the NF1-Schwann cell morphology dataset. 
+# If you would like the more information about this dataset, you can refer to this [repository](https://github.com/WayScience/Benchmarking_NF1_data)
+# 
+# From the mentioned repo, we specifically used this [dataset](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/4_processing_features/data/Plate2/CellProfiler) and the associated [metadata](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/3_extracting_features/metadata) to generate the walkthrough. 
```

Comment:
Should the "dataset" used the generate this walkthrough be directed to the SQLite file and not the already processed features?